src/dgipy/integration/clinical_trials.py

```diff
@@ -0,0 +1,79 @@
+"""Integrate data from FDA clinical trials API."""
+
+from regbot.fetch.clinical_trials import StandardAge
+from regbot.fetch.clinical_trials import get_clinical_trials as get_trials_from_fda
+
+
+def get_clinical_trials(terms: list) -> dict:
```

Comment:
Maybe specify that items in the list should be a str?
```suggestion
def get_clinical_trials(terms: list[str]) -> dict:
```